mail-client/clawsker/clawsker-1.3.5.ebuild

```diff
@@ -0,0 +1,37 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit xdg-utils
+
+DESCRIPTION="Applet to edit Claws Mail's hidden preferences"
+HOMEPAGE="https://www.claws-mail.org/clawsker.php"
+SRC_URI="https://www.claws-mail.org/tools/${P}.tar.xz"
+
+LICENSE="GPL-3"
+SLOT="0"
+KEYWORDS="amd64 x86"
```

Comment:
[ekeyword ~all clawsker-1.3.5.ebuild](https://devmanual.gentoo.org/keywording/index.html#keywording-on-upgrades)

Reply:
Yes, that was silly. Lost my abillities...